.github/styles/Google/WordList.yml

```diff
@@ -58,7 +57,6 @@ swap:
   grayed-out: unavailable
   HTTPs: HTTPS
   in order to: to
```

Comment:
```suggestion
  in order to: to
  indices: indexes
```

Reply:
Noticed this one popped up sometimes when I was reading through the docs.
Also `Postgres: PostgreSQL`?

Reply:
Yes, 'in order to' happens a lot 😉  And yes, Postgres should always be PostgreSQL 👍